### ISSUE TITLE
webcamjs on iphone expects flash

### ISSUE THREAD

**Issue by borisreitman** [jhuckaby/webcamjs#143]
All demos don't work on iPhone 5, they expect flash to be installed.

**Comment by jhuckaby** [comment]
iOS doesn't support getUserMedia nor Flash -- one of which must be present for this library to work. Not much I can do -- sorry.

**Comment by positlabs** [comment]
@borisreitman, to get video input on iphone, you will need something like this.

`<input type="file" accept="video/*;capture=camcorder">`

**Comment by jhuckaby** [comment]
Yeah, I've thought about incorporating something like that into Webcam.JS, but with `image/*` as the MIME type.  Dunno how well it would work tho.

**Comment by positlabs** [comment]
Right, image/* would be the equivalent to `snap()`

I used it on a project last year and it seemed to work pretty well. The only exception was that some android devices would take images that were too large, and it would crash the browser. I think this may have been because I was attempting to draw to canvas, but I don't remember exactly.

I think it would make a nice addition to webcamjs, and it probably won't take much to implement. If the browser doesn't support media input, it will fall back to regular file selection.

**Comment by jhuckaby** [comment]
It shall be attempted!

**Comment by zsalab** [comment]
https://github.com/jhuckaby/webcamjs/pull/209
